docs/rendering-tip.md

```diff
@@ -1,2 +1,50 @@
 # Rendering Tips
+Hubble offers the following options to render your visualizations:
+- GIF
+- WebM video
+- PNG sequence
+- JPEG sequence
 
+If you would like to use the exports in a presentation we recommend the following steps:
+
+### GIF
+Usable as is. Feel free to add it to your presentation, notebook, or showcase!
+
+### Convert WebM
+There are plenty of free converters out there. The ones written in this guide will be based off of: https://convertio.co/
+
+**Step 1: Pick your WebM file**
+![Convertio Step 1](../website/static/images/convertio-step1.jpg)
+
+**Step 2: Click the "..." option to open up a menu**
+
+**Step 3: Select the filetype that you want. We recommend GIF/MP4 for use in your presentations**
+
+**Step 4: Click Convert**
+![Convertio Step 2, 3, & 4](../website/static/images/convertio-step2-3-4.jpg)
+
+### Convert JPEG sequence to GIF
+There are plenty of free converters out there. The ones written in this guide will be based off of: https://gifmaker.me/
+
+**Step 1: Untar the file. On Windows a tool like [7-Zip](https://www.7-zip.org/) can be used**
+
+**Step 2: Click on Upload Images**
+![GIFMaker Step 2](../website/static/images/gifmaker-step2.jpg)
+
+**Step 3: Select all the images in the folder that was created**
+
+**Step 4: Open all the images**
+![GIFMaker Step 3 & 4](../website/static/images/gifmaker-step3-4.jpg)
+
+**Step 4a: A preview will be generated at the top. Settings such as order and pacing can be adjusted if anything looks wrong**
+
+**Step 5: Click on "Create GIF Animation"**
+
+![GIFMaker Step 4a & 5](../website/static/images/gifmaker-step4a-5.jpg)
+
+**Step 6: Click on "Download the GIF"**
+
+![GIFMaker Step 6](../website/static/images/gifmaker-step6.jpg)
```

Comment:
These URLs will be converted into HTML <a href=""/>, so they should be relative to where the server serves static assets. For example to link to, http://hubble.gl/images/visgl-logo.png I'd put `![Cool Image](/images/visgl-logo.png)` in the markdown.

Reply:
Done